wrappers/android/gradle/libs.versions.toml

```diff
@@ -0,0 +1,32 @@
+[versions]
```

Comment:
Is this toml style variable collection the accepted best practice nowadays or your personal preference?